Dockerfile

```diff
@@ -15,4 +15,4 @@ ENV ETCD_URLS ""
 ENV TELEGRAM_TOKEN ""
 
 # RUN
-CMD [ "npm", "start" ]
\ No newline at end of file
+CMD [ "npm", "run", "watch:dev" ]
```

Comment:
@gutorc92, It would be better for CMD to call a custom initialization script. Such script should then decide, based on ENV variables, if the application will run with debug activated or not.